build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -671,6 +674,9 @@ class SeleniumWebDriverController {
     return this.evaluate(() => document.documentElement);
   }
 
+  /**
+   * @return {void}
```

Comment:
```suggestion
   Shutdown the driver.
```

Reply:
This one needed an explicit return type. Added the summary line though.